WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -17,17 +20,140 @@ extension WPStyleGuide
         return [
             NSParagraphStyleAttributeName : paragraphStyle,
             NSFontAttributeName : font,
-            NSForegroundColorAttributeName: WPStyleGuide.grey(),
-        ] as NSDictionary
+            NSForegroundColorAttributeName: grey(),
+        ]
     }
 
     public class func siteAttributionParagraphAttributes() -> NSDictionary {
         let attributes = NSMutableDictionary(dictionary: originalAttributionParagraphAttributes())
-        attributes.setValue(WPStyleGuide.mediumBlue(), forKey: NSForegroundColorAttributeName)
+        attributes.setValue(mediumBlue(), forKey: NSForegroundColorAttributeName)
         return attributes
     }
 
     public class func originalAttributionFontSize() -> CGFloat {
-        return UIDevice.isPad() ? CGFloat(16.0) : CGFloat(14.0)
+        return UIDevice.isPad() ? 16.0 : 14.0
+    }
+
+
+    // MARK: - Reader Card Styles
+
+    // MARK: - Custom Colors
+    public class func readerCardCellBorderColor() -> UIColor {
+        return UIColor(red: 215.0/255.0, green: 227.0/255.0, blue: 235.0/255.0, alpha: 1.0)
+    }
+
+    public class func readerCardCellHighlightedBorderColor() -> UIColor {
+        // #87a6bc
+        return UIColor(red: 135/255.0, green: 166/255.0, blue: 188/255.0, alpha: 1.0)
+    }
+
+    // MARK: - Attributed Text Attributes
+
+    public class func readerCardTitleAttributes() -> [NSObject: AnyObject] {
+        let fontSize:CGFloat = UIDevice.isPad() ? 24.0 : 16.0
```

Comment:
(Maybe?) we could move all of the constants to `private let` definitions, at the bottom of the class?. That way, if we ever need to maintain this one, it'll be in a single spot.